### PR TITLE
stage: move sandbox-cli rbac to infra-deployments

### PR DESCRIPTION
**Pull request by MatousJobanek**
This is another PR part of the transition of all manifests from sandbox-sre into infra-deployments. This time it's about user-management manifests generated by sandbox-cli.
Sandbox-cli generates the manifests based on sandbox-config.yaml file for the given environment. In the context of Konflux, it's mainly about ServiceAccounts that are used by the admins to easily approve signups, move workspaces between the clusters, etc..
Please, be aware that this is "move only"! These manifests are already present in the clusters, so it's about moving the management of the manifests. 
The following PRs can limit the number of users with the given permissions.

### CODE REVIEW

**Comment by MatousJobanek** [comment]
cc @mfrancisc @dperaza4dustbit @sadlerap @filariow  @alexeykazakov

**Review by mfrancisc — APPROVED** [review]
LGTM

Thanks!

**Comment by mfrancisc** [comment]
@hugares you may want to take a look at this as well.

File: components/sandbox/user-management/staging/generated-manifests/host/kustomization.yaml
Comment (hugares) [inline components/sandbox/user-management/staging/generated-manifests/host/kustomization.yaml:1]: You did not include this folder in the host application, is this intentional?

**Review by xcoulon — APPROVED** [review]

File: components/sandbox/user-management/staging/generated-manifests/host/kustomization.yaml
Comment (MatousJobanek) [reply components/sandbox/user-management/staging/generated-manifests/host/kustomization.yaml:1]: it's included here :thinking: 
https://github.com/redhat-appstudio/infra-deployments/blob/cb75811f4356437b2d3b2a123b18806f6a615990/components/sandbox/toolchain-host-operator/staging/stone-stg-host/kustomization.yaml#L9

Comment (hugares) [reply components/sandbox/user-management/staging/generated-manifests/host/kustomization.yaml:1]: sorry for noise, my bad. I did a search but I did not realize that a was not at the root of sandbox folder....

**Review by hugares — APPROVED** [review]

**Pull request merged**